packages/sdk/src/fetcher/subgraph/index.ts

```diff
@@ -30,7 +30,7 @@ import {
 } from "./queries";
 import { ChainId, SUBGRAPH_URL, CHAIN_ADDRESSES } from "../../commons";
 import { enforce } from "../../utils";
-import { getAddress } from "@ethersproject/address";
+import { Address, getAddress } from "viem";
```

Comment:
`Address` can be imported as `type`.

Reply:
Addressed in 9fd53ba9995df6073b3b68b34831a3302c465065